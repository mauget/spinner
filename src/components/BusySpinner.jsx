import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';

function BusySpinner() {
    return (
        <div className="spinner-border" role="status">
            <span className="sr-only">Loading...</span>
        </div>
    );
}
export default BusySpinner;
