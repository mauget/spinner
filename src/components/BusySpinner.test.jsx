import React from 'react';
import BusySpinner from './BusySpinner';
import { shallow } from 'enzyme';

describe('Spinner', ()=>{
    it('should render', ()=>{
        const wrapper = shallow(<BusySpinner />);
        expect(wrapper).toMatchSnapshot();
    });
});
