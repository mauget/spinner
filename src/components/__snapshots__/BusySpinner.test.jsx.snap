// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render 1`] = `ShallowWrapper {}`;
